plugin-infra/go-plugin-infra/test/com/thoughtworks/go/plugin/infra/DefaultPluginManagerTest.java

```diff
@@ -416,6 +419,17 @@ public void shouldThrowExceptionIfMatchingExtensionVersionNotFound() throws Exce
         }
     }
 
+    @Test
+    public void shouldAddPluginChangeListener() throws Exception {
+        DefaultPluginManager pluginManager = new DefaultPluginManager(monitor, registry, mock(GoPluginOSGiFramework.class), jarChangeListener, applicationAccessor, pluginWriter, pluginValidator, systemEnvironment, pluginsZipListener);
+        pluginManager.startInfrastructure();
+
+        InOrder inOrder = inOrder(monitor);
+
+        inOrder.verify(monitor).addPluginJarChangeListener(jarChangeListener);
+//        inOrder.verify(monitor).addPluginJarChangeListener(pluginsZipUpdater);
```

Comment:
Oops. Yeah. The behaviour changed. That line is no longer required :)
Will delete that.